docs/content/modeling/roles-and-permissions.mdx

```diff
@@ -21,16 +21,16 @@ import {
 
 <DocumentationNotice />
 
-In this guide you will learn how to model roles and permissions model within <ProductName format={ProductNameFormat.ProductLink}/> using the _<ProductConcept section="what-is-an-authorization-model" linkName="authorization model" />_ and _<ProductConcept section="what-is-a-relationship-tuple" linkName="relationship tuple" />_.
+This guide explains how to model roles and permissions in <ProductName format={ProductNameFormat.ProductLink}/> using <ProductConcept section="what-is-an-authorization-model" linkName="authorization models" /> and <ProductConcept section="what-is-a-relationship-tuple" linkName="relationship tuples" />. For the purpose of this guide:
 
-- **Roles** are assigned to <ProductConcept section="what-is-a-user" linkName="users" /> or a group of users, where any user can have more than one role (`editor`, `owner`, etc..).
-- **Permissions** are what allows users to access certain <ProductConcept section="what-is-an-object" linkName="objects" /> based on their specific roles (`device_renamer`, `channel_archiver`, etc..).
+- **Roles** are assigned to <ProductConcept section="what-is-a-user" linkName="users" /> or groups of users, where any user can have more than one role, like `editor` or `owner`.
```

Comment:
I think bold UI elements are intended specifically for UI text:
https://oktawiki.atlassian.net/wiki/spaces/DOCS/pages/2544471330/Other+Conventions#UI-text-styling

Reply:
This presumes that we're using a UI, and for now, we're just using CLI. I want to address this in an FGA-specific style guide, but my frame for this was "it's a core product feature that'll be featured in a UI once we build one".

Thoughts?

Reply:
I want to develop this further in an FGA-specific style guide, but: I think bold makes sense in this context because it's a core product feature that _would_ be bold if we had a UI instead of being fully CLI. Thoughts?